core/control-plane/control-plane-aggregate-services/src/main/java/org/eclipse/edc/connector/controlplane/service/ControlPlaneServicesExtension.java

```diff
@@ -0,0 +1,137 @@
+/*
+ *  Copyright (c) 2022 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Bayerische Motoren Werke Aktiengesellschaft (BMW AG) - initial API and implementation
+ *
+ */
+
+package org.eclipse.edc.connector.controlplane.service;
```

Comment:
Can we move this to `org.eclipse.edc.connector.service`? We don't have a `controlplane` package yet. Any control plane functionality is just located at `connector`.

Reply:
See this as a comment for any other following file :D

Reply:
@juliapampus uhm... we have `org.eclipse.edc.connector.dataplane`, is that acceptable?
Will do it anyway.

Reply:
Yes, because it's the explicit name of a framework ("dataplane framework" etc.). The control plane, IMO, is just a theoretical concept. We could also introduce 2 or 3 "planes" that handle different tasks. The only thing they have in common is that they can be assigned to the connector. But these are just my thoughts I had during the package refactoring.

Reply:
I'm not really on the same page on this but for the moment this is not impactful.

Reply:
I am wondering why you did not provide this feedback during the review of my PR last week...

Reply:
you know, things get figured out on the way :')